package.json

```diff
@@ -16,7 +16,8 @@
     ".": {
       "browser": "./dist/trace-mapping.umd.js",
       "require": "./dist/trace-mapping.umd.js",
-      "import": "./dist/trace-mapping.mjs"
+      "import": "./dist/trace-mapping.mjs",
+      "types": "./dist/types/trace-mapping.d.ts"
```

Comment:
Looking at the [docs](https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing), apparently this needs to be first.